src/mobility/components/onboarding/LocationBlocker.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {useCallback} from 'react';
+import {useGeolocationContext} from '@atb/GeolocationContext';
+import {FullScreenView} from '@atb/components/screen-view';
+import {OnboardingScreenComponent} from '@atb/onboarding';
+import {MyLocation} from '@atb/assets/svg/color/images';
+import {MobilityTexts} from '@atb/translations/screens/subscreens/MobilityTexts';
+import {useTranslation} from '@atb/translations';
+
+export type LocationBlockerProps = {};
+
+export const LocationBlocker = ({}: LocationBlockerProps) => {
+  const {t} = useTranslation();
+
+  const {requestLocationPermission} = useGeolocationContext();
+
+  const setCurrentLocationOrRequest = useCallback(async () => {
+    await requestLocationPermission();
+  }, [requestLocationPermission]);
```

Comment:
Note that this might often be blocked, since the user has not accepted it in the normal onboarding. Should probably help the user navigate to the settings.

Reply:
Not sure what you are refering to. This will open the settings dialog if the user already has declined it in the past

Reply:
Great! Didn't remember that.